MonthCellCustomization/MonthCellCustomization/Behavior/CalendarBehavior.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Syncfusion.Maui.Calendar;
+
+namespace MonthCellCustomization
+{
+    public class CalendarBehavior : Behavior<SfCalendar>
+    {
+        private SfCalendar calendar;
+
+        protected override void OnAttachedTo(SfCalendar bindable)
+        {
+            base.OnAttachedTo(bindable);
+            this.calendar = bindable;
+            CalendarTextStyle textStyle = new CalendarTextStyle()
```

Comment:
Assign directly

Reply:
Directly assigned the textstyle instead of using the instance variable